lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -51,32 +51,11 @@ function wp_theme_has_theme_json() {
 if ( ! function_exists( 'wp_theme_has_theme_json_clean_cache' ) ) {
 	/**
 	 * Function to clean the cache used by wp_theme_has_theme_json method.
-	 */
-	function wp_theme_has_theme_json_clean_cache() {
-		wp_cache_delete( 'wp_theme_has_theme_json', 'theme_json' );
-	}
-}
-
-if ( ! function_exists( '_wp_theme_has_theme_json_clean_cache_upon_upgrading_active_theme' ) ) {
-	/**
-	 * Private function to clean the cache used by wp_theme_has_theme_json method.
 	 *
-	 * It is hooked into the `upgrader_process_complete` action.
-	 *
-	 * @see default-filters.php
-	 *
-	 * @param WP_Upgrader $upgrader Instance of WP_Upgrader class.
-	 * @param array       $options Metadata that identifies the data that is updated.
+	 * Not to backport to core. Delete it instead.
 	 */
-	function _wp_theme_has_theme_json_clean_cache_upon_upgrading_active_theme( $upgrader, $options ) {
-		// The cache only needs cleaning when the active theme was updated.
-		if (
-			'update' === $options['action'] &&
-			'theme' === $options['type'] &&
-			( isset( $options['themes'][ get_stylesheet() ] ) || isset( $options['themes'][ get_template() ] ) )
-		) {
-			wp_theme_has_theme_json_clean_cache();
-		}
+	function wp_theme_has_theme_json_clean_cache() {
```

Comment:
This was introduced in 14.5 at https://github.com/WordPress/gutenberg/pull/45543 so I'd rather deprecate it to be safe.